papers/Mathematics/paper.Rmd

```diff
@@ -0,0 +1,269 @@
+outliers
+```
+
+The row numbers of the detected outliers can be obtained by using `which()` on the output object, which can be used for exclusions for example:
+
+```{r}
+which(outliers)
+
+data_clean <- data[-which(outliers), ]
+```
+
+All `check_outliers()` output objects possess a `plot()` method, meaning it is also possible to visualize the outliers:
+
+```{r univariate, fig.cap = "Visual depiction of outliers using the robust z-score method."}
+library(see)
+
+plot(outliers)
+```
+
+Other univariate methods are available, such as using the interquartile range (IQR), or based on different intervals, such as the Highest Density Interval (HDI) or the Bias Corrected and Accelerated Interval (BCI). These methods are documented and described in the function's [help page](<https://easystats.github.io/performance/reference/check_outliers.html>).
+
+## Multivariate Outliers
+
+Univariate outliers can be useful when the focus is on a particular variable, for instance the reaction time, as extreme values might be indicative of inattention or non-task-related behavior^[ Note that they might not be the optimal way of treating reaction time outliers [@ratcliff1993methods; @van1995statistical]].
+
+However, in many scenarios, variables of a data set are not independent, and an abnormal observation will impact multiple dimensions. For instance, a participant giving random answers to a questionnaire. In this case, computing the _z_ score for each of the questions might not lead to satisfactory results. Instead, one might want to look at these variables together.
+
+One common approach for this is to compute multivariate distance metrics such as the Mahalanobis distance. Although the Mahalanobis distance is very popular, just like the regular _z_ scores method, it is not robust and is heavily influenced by the outliers themselves. Therefore, for multivariate outliers, it is recommended to use the Minimum Covariance Determinant, a robust version of the Mahalanobis distance [MCD, @leys2018outliers; @leys2019outliers]. 
+
+In *{performance}*'s `check_outliers()`, one can use this approach with `method = "mcd"`.^[Our default threshold for the MCD method is defined by `stats::qchisq(p = 1 - 0.001, df = ncol(x))`, which again is an approximation of the critical value for _p_ < .001 consistent with the thresholds of our other methods.]
+
+```{r multivariate, fig.cap = "Visual depiction of outliers using the Minimum Covariance Determinant (MCD) method, a robust version of the Mahalanobis distance."}
+results <- check_outliers(data, method = "mcd")
+results
+
+plot(results)
+```
+
+Other multivariate methods are available, such as another type of robust Mahalanobis distance that in this case relies on an orthogonalized Gnanadesikan-Kettenring pairwise estimator [@gnanadesikan1972robust]. These methods are documented and described in the function's [help page](https://easystats.github.io/performance/reference/check_outliers.html).
```

Comment:
Since I don't have enough expert knowledge in that area, I try to stick to recommendations, and I might be wrong but if one method is considered better in 99% of cases then I think it makes sense to only show the recommended method, even if other methods are available. The question is: are some of those other methods better in > 1% of cases? If so, then, yeah maybe we should. But I don't know. My interpretation of Leys et al., is that they aren't.

Also, I added this sentence just to make it consistent with Dom's addition mentioning other univariate methods. I don't know their use cases though, so I would not be able to elaborate on that. @DominiqueMakowski do you feel knowledgeable enough to elaborate on each method's use case?

Reply:
This is not critical. If Dom has no other suggestions we can leave it as is

Reply:
Bump @DominiqueMakowski